### ISSUE TITLE
When Mailcleaner is used as a relay, all BCC list is included in the header

### ISSUE THREAD

**Issue by nomadturk** [MailCleaner/MailCleaner#248]
This line on exim_stage1.conf

```
add_header = :after_received:X-MailCleaner-recipients: $recipients
```

is the perfect line to comment out... When you use Mailcleaner as a relay, this line inserts all of your BCC contacts to headers.

https://forum.mailcleaner.org/viewtopic.php?f=2&t=3090&p=12867&hilit=X+MailCleaner+recipients#p12867

**Comment by kkloesener** [comment]
seems to have been fixed in #280

**Comment by p-rolle** [comment]
The first fix was only for incoming mails.
This issue is now solved in #284
